hkdf.js

```diff
@@ -39,7 +39,12 @@ function hkdf (salt, inputKeyMaterial, info = '', length = 2 * HASHLEN) {
       T[i + 1] = hmacDigest(key, toHash)
     }
 
-    const result = b4a.concat(T.slice(1))
+    const resultWithSlab = b4a.concat(T.slice(1))
```

Comment:
There might be a more efficient way to get T.slice(1) into an allocUnsafeSlow buffer, open to suggestions (or feel free to push to the branch too)

Reply:
Updated to be more efficient